[docs] Update agent init docs for new interactive flow, skill locations, and dotnet-inspect skill (#827)
microsoft/aspire.dev: src/frontend/src/content/docs/get-started/ai-coding-agents.mdx

   aspire agent init
   ```

> 1. Select the **skill locations** where you want files installed. All locations are pre-selected by default:

Reviewer: ```suggestion
1. Select the **skill locations** where you want files installed. The **Standard** location is pre-selected by default:
```

   | Location | Directory | Notes |
   |---|---|---|
   | **Standard** | `.agents/skills/` | Supported by VS Code, GitHub Copilot, and OpenCode |
   | **Claude Code** | `.claude/skills/` | Claude Code specific |
   | **GitHub Skills** | `.github/skills/` | VS Code / GitHub Copilot specific |
   | **OpenCode** | `.opencode/skill/` | OpenCode specific |

1. Select the **skills and tools** to install into those locations. All options are pre-selected by default:

   - **Aspire skill** — teaches your AI agent how to use Aspire CLI commands

>    - **Playwright CLI** — enables browser automation for testing web resources

Reviewer: We need to add the `aspireify skill` too, assuming there's not a PR for it already.

- Add integrations with `aspire add` and search documentation with `aspire docs`
- Use resource MCP tools for database queries and other resource-specific operations

### dotnet-inspect skill

The `dotnet-inspect` skill teaches your AI agent to query .NET API surfaces using the [`dotnet-inspect`](https://github.com/richlander/dotnet-inspect) tool. It's installed alongside the Aspire skill in each selected location:

<FileTree>

- .agents/skills/dotnet-inspect/
  - SKILL.md
- .github/skills/dotnet-inspect/
  - SKILL.md
- .claude/skills/dotnet-inspect/
  - SKILL.md
- .opencode/skill/dotnet-inspect/
  - SKILL.md

</FileTree>

The skill enables your agent to inspect NuGet package API surfaces, compare API changes between package versions, and explore .NET types and members.

> 

Reviewer: ```suggestion

:::note[Skill conflict]
The `dotnet-inspect` skill should not be installed alongside the Aspire skill. The Aspire skill already includes support for `aspire docs api` subcommands, providing a powerful way to query Aspire APIs across both C# and TypeScript.
:::

```
